app/models/user.rb

```diff
@@ -1,11 +1,20 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  include PgSearch
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable, :rememberable, :validatable,
          :omniauthable, omniauth_providers: [:facebook]
   enum favorite_cause: Organization.org_types
+  multisearchable against: %i[first_name last_name nick_name email location],
+                  update_if: %i[
+                    first_name_changed?
+                    last_name_changed?
+                    nick_name_changed?
+                    email_changed?
```

Comment:
Prob don't want search on email, could be used to infer emails which would be a security breach

Reply:
removed